src/App.tsx

```diff
@@ -1,7 +1,23 @@
 import './App.scss';
+import { Outlet } from 'react-router-dom';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { GeneralProvider } from './helpers/GeneralContext';
 
 export const App = () => (
-  <div className="App">
-    <h1>React Phone Catalog</h1>
-  </div>
+  <GeneralProvider>
```

Comment:
Don't you find the name 'GeneralProvider' like... too general? :) 
It's not a mistake, but why don't we think about some alternative naming?